Components/Entities/TransactionalFooter.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Newtonsoft.Json;
```

Comment:
This class seems superfluous, we'll talk :)

Reply:
Removed